src/elements/content-picker/ContentPicker.js

```diff
@@ -1282,12 +1285,14 @@ class ContentPicker extends Component<Props, State> {
                             cancelButtonLabel={cancelButtonLabel}
                             renderCustomActionButtons={renderCustomActionButtons}
                         >
-                            <OffsetBasedPagination
-                                offset={offset}
-                                onOffsetChange={this.paginate}
-                                pageSize={currentPageSize}
-                                totalCount={totalCount}
-                            />
+                            {isPaginationVisible && (
```

Comment:
Slight preference for `{isPaginationVisible ? (...) : null}`. isPaginationVisible is documented as a boolean so the user shouldn't pass in any other values, but it would be nice to cover the possible off-chance that a user passes in a different falsy value. (E.g. if a user decided to pass in `isPaginationVisible=0`, the && condition would display a `0`)

Reply:
Good catch! Will update